iroh/src/commands/start.rs

```diff
@@ -169,6 +176,27 @@ fn migrate_flat_store_v0_v1() -> anyhow::Result<()> {
     Ok(())
 }
 
+fn migrate_flat_store_v1_v2() -> anyhow::Result<()> {
```

Comment:
Not quite sure where to put it. Can you describe what you would want them for?

I don't want to expose details about previous versions of our database too much. That is also why in IrohPaths there is no BlobStoreV0 or BlobStoreV1, but these paths are hardcoded as strings.

Reply:
this can be a follow up PR (but ideally part of this release)

I am not sure where exactly to put this, but `blobstorev1` paths have been used for example in the ffi, so that code needs the migration as well

Reply:
ref: https://github.com/n0-computer/iroh-ffi/blob/main/src/node.rs#L216

Reply:
So the ffi has it's own iroh node? Yeah, in this case I can see that it needs to be exposed. I am sure there is a good reason, but why does iroh-ffi not include the iroh node?

Reply:
> iroh-ffi not include the iroh node?

how would it do that? the `Node::builder` is the expected way to run an iroh node from code